dll/win32/kernel32/client/time.c

```diff
@@ -140,6 +140,16 @@ GetSystemTimeAsFileTime(OUT PFILETIME lpFileTime)
     lpFileTime->dwHighDateTime = SystemTime.HighPart;
 }
 
+/*
+ * @unimplemented
+ */
+VOID
+WINAPI
+GetSystemTimePreciseAsFileTime(OUT PFILETIME lpFileTime)
+{
+    STUB;
```

Comment:
@learn-more, is this actual stub worth having (yet)? Or would using `stdcall -stub` be enough?